docs/UserGuide.md

```diff
@@ -205,9 +205,9 @@ Imports all persons and their details from a CSV file of your specification.
 
 </box>
 
-**Example:** `import i|C:/Users/soojo/Desktop/sample_data.csv` 
+**Example:** `import i|/Users/johansoo/Desktop/AvengersAssemble/exam_data.csv`
```

Comment:
Good attention to detail where matching the filepaths will definitely minimize confusion.